test/SILGen/properties_swift4.swift

```diff
@@ -0,0 +1,126 @@
+
+// RUN: %target-swift-frontend -swift-version 4 -module-name properties -Xllvm -sil-full-demangle -parse-as-library -emit-silgen -disable-objc-attr-requires-foundation-module %s | %FileCheck %s
```

Comment:
You don't use `@objc` in this test, so you don't need `-disable-objc-attr-requires-foundation-module`.